scripts/cleanup.sh

```diff
@@ -0,0 +1,24 @@
+PS4="$PS4"
```

Comment:
`scripts/cleanup.sh` is a super generic name, we should use something more specialized if it's going to live in `scripts/`.  Moving this file to `net/remote/cleanup.sh` could be an alternative.   Also this script needs a shebang